libs/pika/async_mpi/include/pika/async_mpi/trigger_mpi.hpp

```diff
@@ -59,7 +59,10 @@ namespace pika::mpi::experimental::detail {
 
 #if defined(PIKA_HAVE_STDEXEC)
         using completion_signatures = pika::execution::experimental::completion_signatures<
-            pika::execution::experimental::set_value_t()>;
+            pika::execution::experimental::set_value_t(),
+            pika::execution::experimental::set_error_t(std::exception_ptr),
```

Comment:
Also same comment here about forwarding other error types. I can take care of this separately.